### ISSUE TITLE
COIN narration performance

### ISSUE THREAD

**Issue by zhangyl4** [showlab/videollm-online#37]
Hi, first of all, this is an excellent piece of work.

I’ve followed the method outlined in issue #18  to generate the COIN narration training and testing data. Could you kindly provide the performance results of Video LLM Online (Live1 and Live1+) on the test set using only the COIN narration training data? This would help me verify if I’ve correctly generated the data and trained the model as expected.

Thank you!
